process faults {4: ({2, 4, 0, 1}, 4067253202)}
network faults {4: [{5, 1, 0}, {6, 2}, {3}, {4}]}
{6: ("85F9D3F45332B32ED5F1B3B3AD351D805A568C9529E4DAB7D48A47755B886DD7", {0, 4, 5, 2, 1, 6, 3}), 7: ("D383195F70CD6DC4EB6588A64C391B93CF5208A60F0D509F332E2578265A7406", {5, 3, 1, 2, 6, 0, 4}), 5: ("75995F54C2BB324505412654C3E95B2B76062E74ADA331A097ECF4AB91CE54FE", {5, 1, 0, 4, 6, 3, 2}), 9: ("2FA9919557234E60317330D3107EDDBA5059774B26F40F03A61FE79B11298D2A", {4, 1, 5, 0, 3, 6, 2}), 8: ("6866682C22009769020BADC14E57BD03812A0CEA94D12162798CF0F7503FA0B5", {4, 6, 3, 2, 0, 1, 5}), 10: ("3C70CDC920FA3D031AFDE3FFB20C4F16EA7BCC0884B8AF0C5BAD3EB60E7D447C", {2, 5, 3, 0, 4, 1, 6})}
7
reason: all committed
done!
